src/include/ipc/fw-ready-metadata.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ *
+ * Author: Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+ */
+
+#ifndef __IPC_FW_READY_METADATA_H__
+#define __IPC_FW_READY_METADATA_H__
+
+#include <ipc/info.h>
+
+extern const struct sof_ipc_cc_version cc_version;
+extern const struct sof_ipc_probe_support probe_support;
```

Comment:
This shouldn't be in ABI headers. Interface shouldn't define what symbols are in your binary (it's implementation detail at this level).

What's worse we already have something like `extern const struct sof_ipc_cc_version cc_version;` merged in `ipc/info.h`, need to fix it also.

Reply:
Moved `fw-ready-metadata.h` to `include/sof` instead of `include/ipc`